README.md

```diff
@@ -47,10 +54,8 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Thanks to [Ali](https://github.com/ichala) who created the api.
+- Thanks to [Manel](https://github.com/ha-manel/Football-Leagues) for using her project as inspiration.
```

Comment:
- [ ] Please ensure to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).